cmd/build.go

```diff
@@ -52,16 +52,25 @@ func newBuildCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	// buildCmd provides the ability run local builds, or setup/delete Tekton builds, for an appsody project
 	var buildCmd = &cobra.Command{
 		Use:   "build",
-		Short: "Locally build a docker image of your appsody project",
-		Long:  `This allows you to build a local Docker image from your Appsody project. Extract is run before the docker build.`,
+		Short: "Locally build a container image of your Appsody project",
+		Long: `Build a local container image of your Appsody project. The stack along with your Appsody project is first extracted to a local directory, before the docker build is run.
```

Comment:
Suggest a couple of commas, and "first" is a bit unnecessary as we also say "before": 
"The stack, along with your Appsody project, is extracted to a local directory before the docker build is run."